### PR TITLE
CP-49928/`static-vdis`: Add tests (1/3 of: prepare move to `python3/`)

### PR DESCRIPTION
**Pull request by bernhardkaindl**
Summary:

- CP-49928: Cover (test) the functions `list_vdis()` and `fresh_name()` of `static-vdis`
- This is a pre-condition to apply typing fixes to them (to satisfy diff-cover checks in CI)
  - Details moved to my 1st comment below.

Adding just for information: @ashwin9390, @stephenchengCloud

### CODE REVIEW

**Review by psafont — APPROVED** [review]

**Review by ashwin9390 — APPROVED** [review]

**Review by bernhardkaindl — COMMENTED** [review]
- Added `mv` of `test_static_vdis.py` to `python3/tests/`
- Applied using pytest infrastructure to simplify the test cases a lot.
- Rebased to the latest tip of `feature/py3` with race fixes for CI and two python moves merged.

I'll be on PTO until Monday, please merge when a 2nd maintainer approves it.
- This provides the groundwork (pre-conditions) for fixing the pytype and pyright warnings of static-vdis in #5731:
  - After this is done, the next #5731 can be reviewed, or its 2nd commit can be split from it
    - After this is done, the `mv` of `static-vdis` to `python3/` should be possible to do.

**Pull request merged**
